### ISSUE TITLE
Running on a separate machine from Home Assistant using NCS2

### ISSUE THREAD

**Issue by MostHated** [dcmartin/motion-ai#34]
Hey there,
    I just wanted to check and make sure before I got to into it, that this doesn't require the motion-ai portion to be installed on the same host as Home Assistant? I only run the basic install of HA on my Pi4, but all my services I run on a separate Ubuntu VM running on my Unraid server. I was planning on plugging an intel NCS2 into the server and passing it through to the VM. Is this type of setup supported as long as I have the appropriate addons installed in Home Assistant?

Thanks,
-MH

**Comment by dcmartin** [comment]
Yup; that works; you can also run the AI on separate machine; only have to share the MQTT broker
